package.json

```diff
@@ -2,6 +2,10 @@
   "name": "@brightside/core",
   "version": "2.0.0-next.201901171617",
   "description": "Zowe CLI is a command line interface (CLI) that provides a simple and streamlined way to interact with IBM z/OS.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zowe/zowe-cli"
```

Comment:
Per npm doc, this should be `https://github.com/zowe/zowe-cli.git`

See https://docs.npmjs.com/files/package.json#repository